apps/templates/content_templates.py

```diff
@@ -506,18 +506,19 @@ def render_content_template(item, template, update=False):
     :param dict template: template
     :return dict: updates to the item
     """
-
+    NEW_TEMPLATE_DATA_IGNORE_FIELDS = TEMPLATE_DATA_IGNORE_FIELDS.copy()
```

Comment:
it shouldn't be capitalized imo, that implies it's a constant, but you modify it later